keyboards/fc660c/config.h

```diff
@@ -21,15 +21,15 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xFEED
+#define VENDOR_ID       0x4853 /* HS */
```

Comment:
I wasn't sure what to put for the Vendor here. The Hasu controller for the HHKB (which currently has Via support) is "HH". Should this be something like "LP"?

Reply:
Since this isn't actually the original FC660C hardware, HS for Hasu is probably fine - you should also update the `MANUFACTURER` and `PRODUCT` in that case.